### DISCUSSION TITLE
ApollosPlayerContainer & Device Notch

### DISCUSSION THREAD

**Discussion by MatthewKyle22** [ApollosProject/apollos-templates#391] [Q&A] [answered]
I'm curious if there will be a push to add margin above the video player so that video content will show and not get cut off in the notch? I've tried to do this myself adding an override to "ui-connected.NodeSingleConnected" with no avail. @conrad-vanl

**Comment by camrun91** [comment]
@redreceipt  May be able to answer this although I think @conrad-vanl has done the most work on the player.

**Comment by redreceipt** [comment]
@MatthewKyle22 I agree we should fix it. Could you get a screenshot and make an issue?

**Reply by MatthewKyle22** [reply]
Issue created - thanks!

**Comment by redreceipt** [comment, marked as answer]
https://github.com/ApollosProject/apollos-templates/issues/395
